examples/app-basic/src/components/AppConfig/AppConfig.tsx

```diff
@@ -0,0 +1,199 @@
+  isApiKeySet: boolean;
+  // An secret key for our custom API.
+  apiKey: string;
+};
+
+interface Props extends PluginConfigPageProps<AppPluginMeta<JsonData>> {}
+
+export const AppConfig = ({ plugin }: Props) => {
+  const s = useStyles2(getStyles);
+  const { enabled, pinned, jsonData } = plugin.meta;
+  const [state, setState] = useState<State>({
+    apiUrl: jsonData?.apiUrl || '',
+    apiKey: '',
+    isApiKeySet: Boolean(jsonData?.isApiKeySet),
+  });
+
+  const onResetApiKey = () =>
+    setState({
+      ...state,
+      apiKey: '',
+      isApiKeySet: false,
+    });
+
+  const onChangeApiKey = (event: ChangeEvent<HTMLInputElement>) => {
+    setState({
+      ...state,
+      apiKey: event.target.value.trim(),
+    });
+  };
+
+  const onChangeApiUrl = (event: ChangeEvent<HTMLInputElement>) => {
+    setState({
+      ...state,
+      apiUrl: event.target.value.trim(),
+    });
+  };
+
+  return (
+    <div className="gf-form-group">
+      {/* ENABLE / DISABLE PLUGIN */}
+      <div>
+        {/* Enable the plugin */}
+        <Legend>Enable / Disable</Legend>
+        {!enabled && (
+          <>
+            <div className={s.colorWeak}>The plugin is currently not enabled.</div>
+            <Button
+              className={s.marginTop}
+              variant="primary"
+              onClick={() =>
+                updatePluginAndReload(plugin.meta.id, {
+                  enabled: true,
+                  pinned: true,
+                  jsonData,
+                })
+              }
+            >
+              Enable plugin
+            </Button>
+          </>
+        )}
+
+        {/* Disable the plugin */}
+        {enabled && (
+          <>
+            <div className={s.colorWeak}>The plugin is currently enabled.</div>
+            <Button
+              className={s.marginTop}
+              variant="destructive"
+              onClick={() =>
+                updatePluginAndReload(plugin.meta.id, {
+                  enabled: false,
+                  pinned: false,
+                  jsonData,
+                })
+              }
+            >
+              Disable plugin
+            </Button>
+          </>
+        )}
+      </div>
+
+      {/* CUSTOM SETTINGS */}
+      <div className={s.marginTopXl}>
+        <Legend>API Settings</Legend>
+
+        {/* API Key */}
+        <Field label="API Key" description="A secret key for authenticating to our custom API">
+          <SecretInput
+            className={s.input}
+            data-testid="api-key"
+            id="api-key"
+            value={state?.apiKey}
+            isConfigured={state.isApiKeySet}
+            placeholder={'Your secret API key'}
+            onChange={onChangeApiKey}
+            onReset={onResetApiKey}
+          />
+        </Field>
+
+        {/* API Url */}
+        <Field label="API Url" description="" className={s.marginTop}>
+          <Input
+            css=""
```

Comment:
`css` should not be needed here

Reply:
Fixed in https://github.com/grafana/grafana-plugin-examples/pull/26/commits/b61684d5c1a0fe65eaa6974ca06302d3f409a3fe.